geode-core/src/main/java/org/apache/geode/distributed/ConfigurationProperties.java

```diff
@@ -1849,19 +1849,7 @@
    * <U>Since</U>: GemFire 8.0
    */
   String USE_CLUSTER_CONFIGURATION = "use-cluster-configuration";
-  /**
-   * The static String definition of the <i>"user-command-packages"</i> property <a
-   * name="user-command-packages"/a>
-   * </p>
-   * <U>Description</U>: A comma separated list of Java packages that contain classes implementing
-   * the <code>CommandMarker</code> interface. Matching classes will be loaded when the VM starts
-   * and will be available in the GFSH command-line utility.
-   * </p>
-   * <U>Default</U>: <code>""</code>
-   * </p>
-   * <U>Since</U>: GemFire 8.0
-   */
-  String USER_COMMAND_PACKAGES = "user-command-packages";
```

Comment:
since this is public. I wonder if we should deprecate it first, but have it as a no-effect config property.